bvm/ballerina-core/src/main/java/org/ballerinalang/util/codegen/PackageInfoReader.java

```diff
@@ -1055,6 +1055,7 @@ private AttributeInfo getDocumentAttachmentAttribute(ConstantPool constantPool,
         return docAttrInfo;
     }
 
+
```

Comment:
Shall we remove this extra line?

Reply:
removed, I do not know how it came there